Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -953,6 +956,8 @@ extension BrowserViewController: UIContextMenuInteractionDelegate {
         pasteMenuChildren.reverse()
       }
       
+      var copyMenu: UIMenu?
```

Comment:
Why optional?

Reply:
I can remove. I was playing with logic updates so no copy action if on ntp for example